mongo-queries.js

```diff
@@ -0,0 +1,70 @@
+const {DatabaseConnector} = require("./database-connector");
+class MongoQueries {
+    constructor() {
```

Comment:
similar to the previous comments, let's pass the DatabaseConnector in as a parameter in the constructor for this